src/utils/constants.ts

```diff
@@ -2,20 +2,13 @@
  *This file contains constants definitions
  */
 
-export const testArray: string[] = ["Pengu", "Saskwatch", "Monkee"];
-
-/*
- * ClassificationLevel is used by FilterRow.tsx and
- * Filter.tsx to track the classificationLevel of a taxon
- * likely to be updated/changed..
- */
-export const enum ClassificationLevel {
-  KINGDOM = 0,
-  PHYLUM = 1,
-  CLASS = 2,
-  ORDER = 3,
-  FAMILY = 4,
-  GENUS = 5,
-  SPECIES = 6,
-  SUBSPECIES = 7,
-}
+export const classificationLevelArray: string[] = [
```

Comment:
this can be fully typed

Reply:
not sure what is missing here since it has string[]

Reply:
this is saying it can be any string. 
ie: 'car' or 'not a taxon'
when it can only be certain classification levels
type TaxonLevel = 'kingdom' | 'phylum' etc

Reply:
https://steveholgado.com/typescript-types-from-arrays/